app/views/layouts/base.html.erb

```diff
@@ -33,18 +30,20 @@
     </script>
     <%= javascript_include_tag "locale/#{FastGettext.locale}/app" %>
     <%= locale_js_tags %>
-    
     <%= yield(:head) %>
```

Comment:
just before the end of the <head> tags we are getting now many <style> tags which I think comes from this yield:

![image](https://github.com/theforeman/foreman/assets/26363699/36e5585f-b4d1-49de-9f6c-54ac22e18778)

as of now, I don't see an impact on existing pages.